docs/conf.py

```diff
@@ -63,6 +64,11 @@
 
 master_doc = 'contents'
 
+# Multiversion settings
+
+smv_tag_whitelist = None  # r'^v0\.4\.6$'
+smv_branch_whitelist = r'^master$'
```

Comment:
Ouch, whitelist and master in the same line.... I hope that will change soon :-(
Unrelated to what you're adding, of course.